### ISSUE TITLE
unclear documentation and interface in ENDF data source

### ISSUE THREAD

**Issue by gonuke** [pyne/pyne#878]
The ENDF data source requires three arguments to specify a nuclide reaction:
* `nuc`
* `rxn`
* `nuc_i`

The documentation for `nuc_i` does not really clarify how it should be used.  Moreover, it doesn't semantically match how most people think of cross-sections.

Digging into the code, it appears that `nuc` is sometimes not a complete identifier for a nuclide (not sure what it is) and that `nuc_i` is that identifier in such cases.  If this is true, then these semantics are perhaps backwards and the `nuc_i` should be the primary key and the the `nuc` value should be an annotation of some kind.

I think this code came from @jdangerx with @scopatz - comments?

Is anyone using this?

See mailing list: https://groups.google.com/forum/#!topic/pyne-users/JmTpXWkeNyA

**Comment by scopatz** [comment]
We are usingthe ENDF data source for xsgen still I think. OR @FlanFlanagan, are we strictly on the ACE data source for xsgen these days?

**Comment by FlanFlanagan** [comment]
We're using the OpenMC datasource mostly. Though, that's mostly just for now. I'd like to make sure we can use the ENDF datasource in the future.

**Comment by jdangerx** [comment]
> Digging into the code, it appears that nuc is sometimes not a complete
identifier for a nuclide (not sure what it is) and that nuc_i is that
identifier in such cases. If this is true, then these semantics are perhaps
backwards and the nuc_i should be the primary key and the the nuc value
should be an annotation of some kind.

After some industrial-strength remembrance, I have a hazy conception that
the `nuc_i` ignores the `m` bit of the `nucname.id` format, and that ENDF
might also ignore that bit? I would not be surprised if I accidentally used
the wrong name in certain situations, though.

This may shed some light on what's going on for someone more knowledgeable
than I am:
https://github.com/pyne/pyne/blob/6de346fd8c98c4527be4709f31a7e3a4b0624179/pyne/endf.pyx#L629-L636

**Comment by scopatz** [comment]
Yeah, @jdangerx - I think you might be right. ENDF might ignore metastables (or IIRC do something weird with them) and we want to be sure that we can access with metastable info.

**Comment by scopatz** [comment]
I am happy to review a PR if a fix comes in.

**Comment by gonuke** [comment]
I can't tell if this is a bug or a documentation problem.  If a bug, we should delay it.  If documentation, we should address it.
